README.md

```diff
@@ -1,2 +1,24 @@
-# climateiq-login
-Sign up and Login flow implemtation for ClimateIQ
+# ClimateIQ Authentication Service
+
+This is the ClimateIQ Authentication Service app.
+
+## Local Dev Server
+
+```bash
+npm run dev -- --host
+```
+
+## Deploy to Firebase Hosting
+
+Install [firebase-cli](https://firebaseopensource.com/projects/firebase/firebase-tools/#installation), then:
+
+```bash
+npm run build
+firebase deploy
```

Comment:
This deploys to GCP?
I'd guess there are some permissions you need to be able to deploy there?

Reply:
It deploys to Firebase Hosting. Generally, permissions flow from the ClimateIQ GCP. If you get an auth error, you should run the command below to login.

Reply:
Presumably there are some IAM permissions you need to be able to run this command? Could be helpful to document or link to relevant firebase docs.

Reply:
Added links to Firebase IAM console and docs.